homeassistant/components/smartthings/manifest.json

```diff
@@ -29,5 +29,7 @@
   "documentation": "https://www.home-assistant.io/integrations/smartthings",
   "iot_class": "cloud_push",
   "loggers": ["pysmartthings"],
-  "requirements": ["pysmartthings==2.7.0"]
+  "requirements": [
```

Comment:
Needs to be on pypi

Reply:
Yes, but this is for testing purposes now